tests/in/operators.wgsl

```diff
@@ -1,3 +1,35 @@
+// Regexes for the literals are taken from the working draft at
```

Comment:
these should just go into `front/wgsl/tests.rs`

Reply:
There's also the `test_tokens` test in `front/wgsl/lexer.rs`. I'm not quite sure which one is more fitting for these test cases, but I'd hazard a guess the lexer one might be the more fitting one. Feel free to correct me though